database/DBCreation.sql

```diff
@@ -0,0 +1,60 @@
+
+CREATE TABLE  Difficulty (
+	DifficultyID int NOT NULL,
+	DifficultyLevel varchar(250) NOT NULL,
+	PRIMARY KEY(DifficultyID)
+);
+CREATE TABLE Recipe (
+	RecipeID INT IDENTITY(1,1) PRIMARY KEY,
+	Name VARCHAR(150) NOT NULL,
+	Description VARCHAR(255),
+	Instructions VARCHAR(255),
+	PrepTime FLOAT,
+	Difficulty INT,
+	FOREIGN KEY (Difficulty) REFERENCES Difficulty(DifficultyID)
+);
+
+CREATE TABLE Coffee (
+  CoffeeID int  IDENTITY(1,1)  NOT NULL,
```

Comment:
Not MySQL syntax, try
```suggestion
  CoffeeID int  NOT NULL,
```